### PR TITLE
Update requirements.txt and requirements_dev.txt

### LINKED ISSUE #59: 'Hashes are required in --require-hashes mode, but they are missing from some requirements' error raised when setting up for development
**Bug description:**
When [setting up for development](https://github.com/Xcov19/covidX#how-to-setup-for-development), the following error is raised:

```
ERROR: Hashes are required in --require-hashes mode, but they are missing from some requirements. Here is a list of those requirements along with the hashes their do
wnloaded archives actually had. Add lines like these to your requirements files to prevent tampering. (If you did not enable --require-hashes manually, note that it
turns on automatically when any package has a hash.)
    pip-tools==5.3.1 --hash=sha256:73787e23269bf8a9230f376c351297b9037ed0d32ab0f9bef4a187d976acc054
In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
    pre-commit from https://files.pythonhosted.org/packages/bc/9e/8c57b0d2de963ca1ec7ac18b100f2b5f20395473e04a55391d6c3a94c08d/pre_commit-2.9.0-py2.py3-none-any.whl#
sha256=4aee0db4808fa48d2458cedd5b9a084ef24dda1a0fa504432a11977a4d1cfd0a (from -r requirements_dev.txt (line 5))
    pylint_django from https://files.pythonhosted.org/packages/1b/1f/f04b6feeee93b953c699ac9dfb40ac5e155b04d693c79db620c74d1dcf45/pylint_django-2.3.0-py3-none-any.wh
l#sha256=770e0c55fb054c6378e1e8bb3fe22c7032a2c38ba1d1f454206ee9c6591822d7 (from -r requirements_dev.txt (line 6))
    black from https://files.pythonhosted.org/packages/dc/7b/5a6bbe89de849f28d7c109f5ea87b65afa5124ad615f3419e71beb29dc96/black-20.8b1.tar.gz#sha256=1c02557aa099101b
9d21496f8a914e9ed2222ef70336404eeeac8edba836fbea (from -r requirements_dev.txt (line 7))
```

**Reproducing the error:**
1. Setup a virtualenv and run: `CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)" python3 -m pip install -r requirements_dev.txt`
2. See error

**Expected behavior**
All requirements properly installed without any errors being raised.

**Desktop:**
 - OS: Windows

### PR DESCRIPTION
**Pull request by milan-tom**
Closes #59 via updating requirements.txt and requirements_dev.txt using `pip-compile --allow-unsafe --generate-hashes <filename>.txt`.

<a href="https://gitpod.io/#https://github.com/Xcov19/covidX/pull/60"></a>

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/60)

### CODE REVIEW

File: requirements_dev.txt
Comment (codecakes) [inline requirements_dev.txt:1]: there isn't much diff/ between dev and requirements file if it produces a compiled version which includes packages from requirements.

The dev tools keep getting updated and maybe not required for all instances while working on codebase. generally useful only when someone is developing and testing a new feature, they'd dev tools packages.

**Review by codecakes — COMMENTED** [review]
I see the comment  "[setting up for development](https://github.com/Xcov19/covidX#how-to-setup-for-development)"
it should be changed to just requirements.txt
and maybe another subsection added like if you want to install dev packages the install requirements_dev.txt like `python3 -m pip install -r requirements_dev.txt`

I think the readme should be updated itself;

File: requirements_dev.txt
Comment (milan-tom) [reply requirements_dev.txt:1]: Okay, I will amend the readme file.

**Comment by milan-tom** [comment]
I have changed th

> I see the comment "[setting up for development](https://github.com/Xcov19/covidX#how-to-setup-for-development)"
> it should be changed to just requirements.txt
> and maybe another subsection added like if you want to install dev packages the install requirements_dev.txt like `python3 -m pip install -r requirements_dev.txt`
> 
> I think the readme should be updated itself;

I have changed and repositioned the command to a position I think makes more sense. I have also added an extra subsection to install developer packages.

File: Readme.md
Comment (codecakes) [inline Readme.md:91]: `CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)"` this part is not needed for dev tools pkg

**Review by codecakes — APPROVED** [review]
looks good. there's a slight one liner change i've added. after that pls merge this.

File: Readme.md
Comment (milan-tom) [reply Readme.md]: I await your approval for the change.

**Pull request merged**
